Userland/Libraries/LibWeb/Bindings/WindowObject.cpp

```diff
@@ -70,6 +72,14 @@ void WindowObject::initialize_global_object()
     define_native_function("atob", atob, 1, attr);
     define_native_function("btoa", btoa, 1, attr);
 
+    define_native_accessor("scrollX", scroll_x_getter, {}, JS::Attribute::Enumerable);
+    define_native_accessor("pageXOffset", scroll_x_getter, {}, JS::Attribute::Enumerable);
+    define_native_accessor("scrollY", scroll_y_getter, {}, JS::Attribute::Enumerable);
+    define_native_accessor("pageYOffset", scroll_y_getter, {}, JS::Attribute::Enumerable);
+    define_native_function("scroll", scroll, 2, attr);
+    define_native_function("scrollTo", scroll, 2, attr);
+    define_native_function("scrollBy", scroll_by, 2, attr);
```

Comment:
We tend to use `vm.names.scrollX` instead of hardcoding but I don't know how that should be done here. Perhaps we should make a LibWeb equivalent. (cc @linusg)

Reply:
Actually the ones above also have hardcoded strings so that may just be how we do things around LibWeb.

Reply:
Yeah, there are a couple of property names specifically for LibWeb in LibJS's CommonPropertyNames, but not many (and they don't belong there either, IMO). Just continue using string literals for now.